web_external/Datasets/ApplyMetadataView.js

```diff
@@ -40,7 +40,7 @@ var MetadataFileCollection = Collection.extend({
     sortDir: SORT_DESC,
 
     parse: function (data) {
-        _.each(data, function (item) {
+        _.each(data, (item) => {
```

Comment:
We could start replacing these with [for...of](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...of). Some other underscore methods could probably disappear as well (_.map?). This can happen in another pass.